### PR TITLE
add : added more necessarys svg's

### PR DESCRIPTION
**Pull request by Shashisxp**
added more necessary svg's to public folder

### CODE REVIEW

File: public/menu.svg
Comment (Ayush-kathayat) [inline public/menu.svg:1]: hey also add close menu svg like the multiply sign and also i don't see the icons like social icons example github , twiiter , linkedin etc

**Comment by Shashisxp** [comment]
now its done merge it

File: public/menu.svg
Comment (Shashisxp) [reply public/menu.svg:1]: done i have added required svgs

**Comment by Ayush-kathayat** [comment]
okay alll good i will give you the next task in the evening good work

**Pull request merged**